packages/fractor-typoscript/src/ValueObject/TypoScriptPrettyPrinterFormatConfiguration.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorTypoScript\ValueObject;
+
+final readonly class TypoScriptPrettyPrinterFormatConfiguration
+{
+    public function __construct(
+        private int $size,
```

Comment:
These could be `public` and we can get rid of the getters then.

Reply:
I'm not a fan of public properties but if you think it's better, you can change it

Reply:
I agree in general, but in readonly classes (i.e. ValueObjects) it's fine IMO.